### ISSUE TITLE
!pip install torchquantum

### ISSUE THREAD

**Issue by mehrnia** [mit-han-lab/torchquantum#286]
why I get error for installing this:
!pip install torchquantum
I installed the dependencies still having error:
  
  -- Configuring incomplete, errors occurred!
  error: command 'C:\\Users\\mma3012\\AppData\\Local\\Temp\\pip-build-env-zvqyulnt\\overlay\\Scripts\\cmake.EXE' failed with exit code 1
  [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyscf
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (pyscf)

**Comment by 01110011011101010110010001101111** [comment]
What environment/device was this run on? WSL?

**Comment by 01110011011101010110010001101111** [comment]
Actually, I recommend checking out this issue: https://github.com/mit-han-lab/torchquantum/issues/266

Feel free to follow up with questions from that thread!

**Comment by mehrnia** [comment]
Hi thank you for following up, I get the issue on windows and running anaconda, powershell and then Spyder.

**Comment by 01110011011101010110010001101111** [comment]
Yeah, in that case, I would recommend to check out the link in the above issue with pyscf. 

Actually, I don’t think pyscf is necessary anymore for torchquantum. I don’t have direct access to the pip deployment, but I’ll open a PR removing pyscf. If you’d prefer, you can install from source and remove the pyscf from the requirements.
